<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ingame</class>
 <widget class="QWidget" name="ingame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1159</width>
    <height>676</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="boardui" name="boardholder" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>681</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="name">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>415</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>nameholder</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_health">
   <property name="geometry">
    <rect>
     <x>556</x>
     <y>450</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>HEALTH</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_sanity">
   <property name="geometry">
    <rect>
     <x>556</x>
     <y>480</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>SANITY</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_chars">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>490</y>
     <width>391</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>BOOK     TRADE     EYES     STRENGTH     WILL</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_book">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>450</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_trade">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>450</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_eyes">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>450</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_strength">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>450</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_will">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>450</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_healthnum">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>440</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_sanitynum">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>470</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_book_impr">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>450</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="l_trade_impr">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>450</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="l_eyes_impr">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>450</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="l_strength_impr">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>450</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="l_will_impr">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>450</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="l_clues">
   <property name="geometry">
    <rect>
     <x>46</x>
     <y>480</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>CLUES</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_cluesnum">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>470</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QListWidget" name="hand_magic">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>530</y>
     <width>151</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="hand_arts">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>530</y>
     <width>141</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="hand_bads">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>530</y>
     <width>151</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="infobox">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>440</y>
     <width>271</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="b_move">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>200</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Move</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_trade">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>240</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Trade</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_ticket">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>280</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Ticket</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_exchange">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>320</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Exchange</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_sleep">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>360</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Sleep</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_ability">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>400</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Ability</string>
   </property>
  </widget>
  <widget class="QComboBox" name="hand_things">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>540</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="list_throws">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>110</y>
     <width>81</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="check_count">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>120</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="reserve">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buy">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>190</y>
     <width>61</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Buy</string>
   </property>
  </widget>
  <widget class="QPushButton" name="drop">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>190</y>
     <width>61</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Drop</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>boardui</class>
   <extends>QWidget</extends>
   <header>board.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
